src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+    public Ball(String color, int number){

```

Comment:
Add a space before the opening brace of the constructor to improve readability. 😊